plugin/PDFView.qml

```diff
@@ -190,7 +198,41 @@ SilicaFlickable {
 
     children: [
         HorizontalScrollDecorator { color: Theme.highlightDimmerColor },
-        VerticalScrollDecorator { color: Theme.highlightDimmerColor }
+        VerticalScrollDecorator { color: Theme.highlightDimmerColor },
+        PDFContextMenu {
```

Comment:
Well, it should be possible to set different parent property value for the menu, but still I think it could be cleaner having the element where it visually should be. View already provides gotoPage() and similar functionality which is used from document page. Only additional requirement would be passing click signals from view.